vulnfeeds/cmd/cperepos/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
+    apt-get install -y \
+    	apt-transport-https \
+	ca-certificates \
+	curl \
+	golang-go
+
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+	curl https://packages.cloud.google.com/apt/doc/apt-key.gpg -o /usr/share/keyrings/cloud.google.gpg && \
+        apt-get update && apt-get install -y google-cloud-sdk
+
+RUN mkdir /src
+WORKDIR /src
+
+COPY ./go.mod /src/go.mod
+COPY ./go.sum /src/go.sum
+RUN go mod download
+
+COPY ./ /src/
+RUN go build -o cperepos ./cmd/cperepos
```

Comment:
I know we are doing this in some other images, but I don't think we need to be building these binaries (and thereby including all the files required/generated by building) inside of our running container.

The [terraform](https://github.com/google/osv.dev/blob/master/docker/terraform/Dockerfile) and [indexer](https://github.com/google/osv.dev/blob/master/docker/indexer/Dockerfile) have multi-stage builds, which I think is better

Reply:
Done in [0fd5da6](https://github.com/google/osv.dev/pull/1146/commits/0fd5da63efcf108cccf1714a78d75cc54769cdb8)